Model/Config.php

```diff
@@ -654,12 +692,32 @@ public function getCheckoutFlowType()
     }
 
     /**
-     * Get partial capture
+     * Get partial capture (US only)
      *
      * @return bool
      */
-    public function getPartialCapture()
+    public function getPartialCapture($countryCode = 'USA')
```

Comment:
Please use global enum/constants instead of hard-coded value

Reply:
replaced hardcoded value throughout the class